### PR TITLE
Add french translation of Twitter page

### PR DESCRIPTION
**Pull request by RedNesto**
Yeah, I like translating things.

_Actually working on GameModes_

### CODE REVIEW

File: wiki/Twitter/fr.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Nous tweetons aussi!
```
Comment (kj415j45) [inline wiki/Twitter/fr.md]: >`# Nous tweetons aussi!`

Comment (RedNesto) [inline wiki/Twitter/fr.md]: I know, but the english file is like this...

File: wiki/Twitter/fr.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+osu! a plusieurs comptes twitter, chacun sert à quelque chose de différent. La plupart ne tweetent que peu mais vous gardent au courant de ce qui se passe dans l'écosystème d'osu!
```
Comment (SpartanPlume) [inline wiki/Twitter/fr.md]: __"La plupart ne tweetent que peu"__ is (maybe?) incorrect, but it's better to say __"La plupart ne tweetent qu'un peu"__.

Comment (RedNesto) [inline wiki/Twitter/fr.md]: Both are valid to me

Comment (SpartanPlume) [inline wiki/Twitter/fr.md]: Oh, I know why it was bothering me. In this case, "peu" is used like a "nom commun", so it needs an "article" ("un") to construct a correct sentence.

Comment (RedNesto) [inline wiki/Twitter/fr.md]: Done

Comment (kj415j45) [inline wiki/Twitter/fr.md]: Actually, you can talk with French, It's ok

File: wiki/Twitter/fr.md (hunk 1 above)
Comment (TPGPL) [inline wiki/Twitter/fr.md]: see my newest pr

Comment (RedNesto) [inline wiki/Twitter/fr.md]: I'll do the change once your pr is merged

File: wiki/Twitter/fr.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+## Nous tweetons aussi!
+
+osu! a plusieurs comptes twitter, chacun sert à quelque chose de différent. La plupart ne tweetent qu'un peu mais vous gardent au courant de ce qui se passe dans l'écosystème d'osu!
+
+Avatar | Compte | Description
+:----: | :----: | -----------
+![osu!status avatar](https://pbs.twimg.com/profile_images/706719922596900864/xTzREmuc_bigger.jpg) | [@osustatus](https://twitter.com/osustatus) | Dédié à fournir de rares notifications sur les problèmes du site et de bancho. Si vous devriez suivre un compte, c'est celui-ci!
```
Comment (tom-arrow) [inline wiki/Twitter/fr.md]: Espace après ❗️

File: wiki/Twitter/fr.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+![osu! avatar](https://pbs.twimg.com/profile_images/706741102120800256/xOf4lxBy_bigger.jpg) | [@osugame](https://twitter.com/osugame) | Un flux des actualités et des notifications des [beatmaps récemment rankées](https://osu.ppy.sh/p/beatmaplist).
+![Dean Herbert avatar](https://pbs.twimg.com/profile_images/646911857802907648/6Ojh9ewB_bigger.png) | [@ppy](https://twitter.com/ppy) | Pas strictement lié à osu!, mais je le laisse ici pour ceux qui sont intéressés. Ma vie c'est osu!, après tout.
+![BanchoBoat avatar](https://pbs.twimg.com/profile_images/1238393475/bboat_bigger.jpg) | [@banchoboat](https://twitter.com/banchoboat) | Un élan comique quand les choses se passent mal
```
Comment (tom-arrow) [inline wiki/Twitter/fr.md]: Point final manquant

**Pull request merged**